### PR TITLE
Make amp-font polyfill use font-weight and font-style to download the right font file

### PR DESCRIPTION
**Pull request by sriramkrish85**
Fixes https://github.com/ampproject/amphtml/issues/3296

### CODE REVIEW

File: extensions/amp-font/0.1/fontloader.js [hunk 1]
```diff
@@ -183,6 +183,8 @@ export class FontLoader {
       // Use larger font-size to better detect font load.
       fontSize: '40px',
       fontVariant: this.fontConfig_.variant,
+      fontWeight: this.fontConfig_.weight,
```
Comment (dvoytenko) [inline extensions/amp-font/0.1/fontloader.js]: To confirm: these are all "normal" by default?

Please also add a test.

Comment (sriramkrish85) [inline extensions/amp-font/0.1/fontloader.js]: Yes these are normal by default - added checks to existing tests

**Comment by sriramkrish85** [comment]
PTAL

**Comment by dvoytenko** [comment]
LGTM

**Pull request merged**